nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/record/script/ScriptedRecordSetWriter.java

```diff
@@ -60,12 +61,16 @@ public void onEnabled(final ConfigurationContext context) {
         super.onEnabled(context);
     }
 
-
     @Override
     public RecordSetWriter createWriter(ComponentLog logger, RecordSchema schema, OutputStream out) throws SchemaNotFoundException, IOException {
+        return createWriter(new HashMap<>(), logger, schema, out);
```

Comment:
`Collections.emptyMap()` would be better than `new HashMap<>()` to avoid unnecessary instance creation.

Reply:
Thanks, Updated.